addons/dropbox/models.py

```diff
@@ -227,7 +227,7 @@ def create_waterbutler_log(self, auth, action, metadata):
         )
 
     def __repr__(self):
-        return u'<NodeSettings(node_id={self.owner._primary_key!r})>'.format(self=self)
+        return f'<NodeSettings(node_id={self.owner._primary_key!r})>'
```

Comment:
Fitz is suspicious...  What is the `!r` magic and does it apply to f-strings the same as u-strings?

Reply:
To be clear: this might be totally fine, but I'm not familiar with it. I'm leaving this comment as a reminder to come back and review this.

Reply:
I checked, this is indeed an issue, I will go over an fix all occurrences

Reply:
Thans for taking notice of that! This is appropriate syntax, as you can see below
https://docs.python.org/3.10/tutorial/inputoutput.html
![image](https://github.com/CenterForOpenScience/osf.io/assets/158075011/3d10d0a2-f828-4736-ac4d-1f698691b3ed)

Reply:
👍